packages/db/package.json

```diff
@@ -12,11 +12,11 @@
     "with-env": "dotenv -e ../../.env --"
   },
   "dependencies": {
+    "prisma": "^4.12.0",
```

Comment:
same here - `prisma` is a devDep, why does it need to be a prod dep now?

Reply:
in the turbo.json we declare that before build it should run `db:generate`, `db:generate` runs `prisma` and because we only install `dependencies` I had to add it.

Reply:
As long as you're using `output: "standalone"` devDep will not be included on the final image, so imo you don't need `--prod` when running `pnpm install`, specially since you're doing `prune` anyway.